aquarius/events/events_monitor.py

```diff
@@ -189,6 +202,16 @@ def thread_process_ve_allocate(self):
                     logger.error(f"Error updating ve_allocate list: {str(e)}.")
             time.sleep(self._monitor_sleep_time)
 
+    def thread_process_nft_ownership(self):
```

Comment:
a couple of suggestions:
would it be possible to place "self._thread_process_nfts_is_on" before the "while"?
also, this function updates ownership but does not return anything, is that ok? it could return a boolean so we can use it as a hook
again, just suggestions...

Reply:
the purpose of having that check in the while loop is that, at some point in future, we can switch on/off threads while aqua is running